api/api/src/main/scala/uk/ac/wellcome/platform/api/models/Aggregation.scala

```diff
@@ -86,8 +86,8 @@ object Aggregations extends Logging {
 
   implicit val decodeLocationTypeFromLabel: Decoder[LocationTypeQuery] =
     Decoder.decodeString.map {
-      case "DigitalLocation"  => LocationTypeQuery.DigitalLocation
-      case "PhysicalLocation" => LocationTypeQuery.PhysicalLocation
+      case "DigitalLocationDeprecated"  => LocationTypeQuery.DigitalLocation
```

Comment:
Are we sure we want this? @jamesgorrie?

Reply:
This should stay the same, I need to do the work to go and get rid of the `Deprecated` stuff.

Reply:
Not sure I understand what the issue with this is :thinking: 

This currently queries the deprecated locations field, but using the `ontologyType` rather than `type`. Can you explain what the problem is?

Reply:
The problem is with adding the `Deprecated` part to the string. It should just be `DigitalLocation` and `PhysicalLocation`.

The `Deprecated` classes are going to be un-deprecated as the work done there was made redundant.

Reply:
At that stage we can remove the `Deprecated` suffix then. For now it is required as that is how they are serialised

Reply:
Yip - Let's do it like that.